modules/ROOT/pages/template.adoc

```diff
@@ -5,6 +5,9 @@
 :pluginname: Template
 :plugincode: template
 
+
+IMPORTANT: The Template plugin, has been marked as *deprecated*. As a result, it will be completely removed from the upcoming TinyMCE 7.0 release. As an alternative solution, we highly recommend utilizing the xref:advanced-templates.adoc[Advanced Template] Premium plugin. 
```

Comment:
```suggestion
IMPORTANT: The Template plugin has been marked as *deprecated*. As a result, it will be completely removed from the upcoming TinyMCE 7.0 release. As an alternative solution, we highly recommend utilizing the xref:advanced-templates.adoc[Advanced Template] Premium plugin. 
```